keyboards/mechlovin/infinity875/info.json

```diff
@@ -5,15 +5,7 @@
     "maintainer": "Mechlovin' Studio",
     "usb": {
         "vid": "0x4D4C",
-        "pid": "0x0875",
-        "device_version": "0.0.1"
     },
-    "indicators": {
-        "caps_lock": "D6",
-        "scroll_lock": "B4"
-    },
-    "processor": "atmega32u4",
-    "bootloader": "atmel-dfu",
     "layouts": {
```

Comment:
There are some layouts here, the common `json` file. But they dont contain matrix information, only x/y coords
Given that rev1 and rev2 dont have such information (neither on their `json` nor the deprecated `.h` files)... does the code even work right now?